tests/e2e/core-tests/specs/index.js

```diff
@@ -63,6 +64,7 @@ const runShopperTests = () => {
 };
 
 const runMerchantTests = () => {
+	runAddShippingClassesTest()
```

Comment:
You also need to export this function at the bottom of the file and create a single test file in `tests/e2e/specs/wp-admin`.

Reply:
Thank you @rrennick.
Done on 8d4ebb935e7807defbf6770e8d7705a6c755275b